servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/CheckContent.java

```diff
@@ -67,7 +67,12 @@ public class CheckContent extends BaseCommand {
   @CommandLine.Option(
       names = {"-B", "--batch"},
       defaultValue = "25",
-      description = "The max number of keys to load at the same time.")
+      description = {
+        "The max number of keys to load at the same time.",
+        "If an error occurs while loading or parsing the values for a single key, the error "
+            + "will be propagated to all keys processed in the same batch. In such a case, rerun "
+            + "the check for the affected keys either with a batch size of 1."
```

Comment:
Nit: the word "either" seems dangling, did you mean "rerun the check for each of the affected keys with a batch size of 1"?